src/Game.qml

```diff
@@ -564,43 +566,45 @@ Item {
         visible: false
         opacity: visible ? 0.8 : 0.0
         scale: visible ? 1 : 0
+        color: "#000000"
         Behavior on scale { NumberAnimation { duration: 200; easing.type: Easing.InCurve } }
-        Text {
+        Label {
             anchors.top: parent.top
             width: parent.width
-            height: parent.height*0.7
-            color: "#776e65"
+            height: parent.height*0.8
             //% "Game Over"
             text: qsTrId("id-game-over")
             font.bold: true
             horizontalAlignment: Text.AlignHCenter
             verticalAlignment: Text.AlignVCenter
-            font.pixelSize: parent.height/text.length
+            font.pixelSize: Dims.l(10)
         }
-        Rectangle {
-            color: "#8f7a66"
-            x: parent.width*0.3
-            y: parent.height*0.5
-            width: parent.width*0.4
-            height: parent.height*0.15
-
-            Text {
-                anchors.top: parent.top
-                width: parent.width
-                height: parent.height
-                color: "#f9f6f2"
+        MouseArea {
```

Comment:
This is a good bit of logic, eventually it would be good to encapsulate that in qml-asteroid, but this is not too urgent right no and can be done another time! :)